autode/neb/original.py

```diff
@@ -531,3 +614,32 @@ def idpp_relax(self) -> None:
 
         self.images.set_coords(result.x)
         return None
+
+    def _max_atom_distance_between_images(self,
+                                          idxs: Optional[Sequence[int]] = None
+                                          ) -> Distance:
+        """
+        Calculate the maximum atom-atom distance between two consecutive images
+        """
+        if idxs is None:  # Use all pairwise distances
+            idxs = np.arange(self.images[0].species.n_atoms)
+        else:
+            idxs = np.array(idxs)
+
+        overall_max_distance = -np.inf
+
+        for i in range(len(self.images) // 2):
+            k = 2 * i
+            k_plus1 = 2 * i + 1
```

Comment:
Passing k + 1 into x_j would be a bit clearer to me

Reply:
_strong_ agree 🤦🏼